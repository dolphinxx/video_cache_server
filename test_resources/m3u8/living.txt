#EXTM3U
#EXT-X-TARGETDURATION:10
#EXT-X-VERSION:4
#EXT-X-MEDIA-SEQUENCE:1
#EXTINF:10.0,
fileSequence1.ts
#EXTINF:10.0,
/fileSequence2.ts
#EXTINF:10.0,
http://media.anywhere.com/fileSequence3.ts
#EXTINF:10.0,
http://media.anywhere.com/stream/fileSequence4.ts
#EXTINF:10.0,
https://media.anywhere.com/stream/fileSequence5.ts
#EXTINF:10.0,
//media.anywhere.com/stream/fileSequence6.ts