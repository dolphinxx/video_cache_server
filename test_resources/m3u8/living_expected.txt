#EXTM3U
#EXT-X-TARGETDURATION:10
#EXT-X-VERSION:4
#EXT-X-MEDIA-SEQUENCE:1
#EXTINF:10.0,
https://test.com/m3u8/fileSequence1.ts
#EXTINF:10.0,
https://test.com/fileSequence2.ts
#EXTINF:10.0,
http://media.anywhere.com/fileSequence3.ts
#EXTINF:10.0,
http://media.anywhere.com/stream/fileSequence4.ts
#EXTINF:10.0,
https://media.anywhere.com/stream/fileSequence5.ts
#EXTINF:10.0,
https://media.anywhere.com/stream/fileSequence6.ts